### ISSUE TITLE
Enable Documentation preview on each PR

### ISSUE THREAD

**Issue by cshilwant** [TexasInstruments/processor-sdk-doc#20]
For each PR, generate equivalent html output and host it somewhere.
This will help in reviewing docs and catching issues

**Comment by cshilwant** [comment]
@StaticRocket @praneethbajjuri Moving the discussion here,

What's your thoughts on https://github.com/TexasInstruments/processor-sdk-doc/pull/126#issuecomment-2544599347

**Comment by StaticRocket** [comment]
I'm not sure we should trust user provided previews, but I don't know.

Also, they'll have to tweak their workflows for that to pickup other branches. It just gets messy.

**Comment by praneethbajjuri** [comment]
Agreed here . Having PR level preview in the main doc project workflow makes sense.

**Comment by cshilwant** [comment]
I see, Fair point!
